app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @theme = @project.challenge_theme
```

Comment:
The @theme instance variable isn't strictly necessary because you can call `@project.challenge.theme` in the view, but it's also okay to encapsulate that call into a model method if that feels cleaner!